### PR TITLE
adding support for new /qiita_db/artifacts

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: .github/workflows/qiita-plugin-ci.yml [hunk 1]
```diff
@@ -176,7 +176,7 @@ jobs:
     - name: flake8
       uses: actions/setup-python@v2
       with:
-        python-version: 3.6
+        python-version: 3.9
```
Comment (charles-cowart) [inline .github/workflows/qiita-plugin-ci.yml:179]: @antgonza is changing to 3.9 okay? I thought plugins required 3.6

**Review by charles-cowart — CHANGES_REQUESTED** [review]
Everything looks good. Just one question needs confirmation before approving.

File: .github/workflows/qiita-plugin-ci.yml (hunk 1 above)
Comment (antgonza) [reply .github/workflows/qiita-plugin-ci.yml:179]: That's _not_ for plugins, that's for flake8 and it needs 3.9.

Comment (charles-cowart) [reply .github/workflows/qiita-plugin-ci.yml:179]: Ok.

**Review by charles-cowart — APPROVED** [review]
Approved.

**Pull request merged**